cachelib/allocator/CacheAllocator-inl.h

```diff
@@ -49,12 +50,16 @@ CacheAllocator<CacheTrait>::CacheAllocator(Config config)
       cacheCreationTime_{util::getCurrentTimeSec()},
       nvmCacheState_{config_.cacheDir, config_.isNvmCacheEncryptionEnabled(),
                      config_.isNvmCacheTruncateAllocSizeEnabled()} {
+  // TODO(MEMORY_TIER)
+  if (memoryTierConfigs.size())
+    throw std::runtime_error("Using custom memory tier is only supported for Shared Memory.");
```

Comment:
for coding standards compliance, I would prefer {} for the if condition statement.